### PR TITLE
build and send delegation transactions

### PR DESCRIPTION
**Pull request by SebastienGllmt**
Haven't tested this yet so probably there are some bugs to fix and there are some TODOs also

### CODE REVIEW

File: app/containers/wallet/staking/SeizaFetcher.js [hunk 1]
```diff
@@ -2,24 +2,31 @@
...
-};
+const useIframeMessageReceiver: ({|
+    createTransaction: PoolRequest => void,
+    setSelectedPools: Array<SelectedPool> => void,
+|}) => void = (payload) => {
+  const messageHandler = (event) => {
+    if (event.origin !== process.env.SEIZA_FOR_YOROI_URL) return;
+    const pools: Array<SelectedPool> = JSON.parse(decodeURI(event.data));
+    payload.createTransaction({
+      // TODO: fix pool ids from Seiza
```
Comment (SebastienGllmt) [inline app/containers/wallet/staking/SeizaFetcher.js]: TODO (Nico will fix this)

File: app/containers/wallet/staking/SeizaFetcher.js [hunk 2]
```diff
@@ -33,17 +40,60 @@ const Staking = (props: {|
...
+    setSelectedPools,
+  });
 
-  useIframeMessageReceiver();
+  const delegationTx = delegationTxStore.createDelegationTx.result;
 
   if (stakingUrl == null) {
     throw new Error('Staking undefined SEIZA_FOR_YOROI_URL should never happen');
   }
+  // TODO: some dialog component that loads when createDelegationTx is executing
```
Comment (SebastienGllmt) [inline app/containers/wallet/staking/SeizaFetcher.js]: TODO (need to fix before merging)

File: app/containers/wallet/staking/SeizaFetcher.js [hunk 3]
```diff
@@ -33,17 +40,60 @@ const Staking = (props: {|
...
   return (
+    <>
+      {delegationTx != null &&
+        <DelegationTxDialog
+          staleTx={delegationTxStore.isStale}
+          poolName={selectedPools[0].name}
+          poolHash={selectedPools[0].poolHash}
+          transactionFee={getShelleyTxFee(delegationTx.IOs, false)}
+          amountToDelegate={delegationTxStore.amountToDelegate}
+          approximateReward="0.0" // TODO
```
Comment (SebastienGllmt) [inline app/containers/wallet/staking/SeizaFetcher.js]: TODO: need to replace with approximation

**Comment by emurgo-rnd** [comment]
test

**Pull request merged**